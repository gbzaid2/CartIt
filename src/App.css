.all-questions {
  max-height: 500px;
  overflow: auto;
  border: 2px;
  border-color: red;
  border-style: dashed;
}

.search-box {
  width: 500px;
  height: 30px;
}