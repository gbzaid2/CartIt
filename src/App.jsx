import React from "react";
import { Route, Switch } from "react-router-dom";
import NavBar from "./components/NavBar.jsx";
import Home from "./components/Home.jsx";
import RatingsAndReviews from "./components/ratings-and-reviews/RatingsAndReviews.jsx";
import Overview from "./components/overview/Overview.jsx";
import QA from "./components/questions-and-answers/QA.jsx";
import RIC from "./components/related-items-and-comparison/RIC.jsx";
import "./App.css";
import RatingSummary from "./components/ratings-and-reviews/RatingSummary.jsx";

const App = (props) => {
  console.log(process.env.REACT_APP_API_URL);
  return (
    <>
      <div>
        <NavBar />
        <section className="section">
          <Switch>
            <Route exact path="/" render={() => <Home />} />
            <Route exact path="/overview" render={() => <Overview />} />
            <Route exact path="/questions-and-answers" render={() => <QA />} />
            <Route exact path="/related-items-and-comparison" render={() => <RIC />} />
            <Route
              exact
              path="/ratings-and-reviews"
              render={() => <RatingsAndReviews />}
            />
          </Switch>
          <Overview />
          <QA />
          <RIC />
          <RatingsAndReviews />
        </section>
      </div>
    </>
  );
};

export default App;
