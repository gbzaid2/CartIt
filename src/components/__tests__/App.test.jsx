import React from "react";
import { shallow } from "enzyme";
import App from "../../App.jsx";
import renderer from "react-test-renderer";

test("Header tag says 'Hello World'", () => {
  const wrapper = shallow(<App />);
  const h1 = wrapper.find("h1");
  expect(h1.text()).toBe("Hello World");
});

test("App component renders a snapshot properly", () => {
  const app = renderer.create(<App />).toJSON();
  expect(app).toMatchSnapshot();
});
