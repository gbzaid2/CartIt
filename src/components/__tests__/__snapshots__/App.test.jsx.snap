// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component renders a snapshot properly 1`] = `
<div>
  <div
    className="container"
  >
    <h1
      className="title is-1 has-text-centered"
    >
      Hello World!
    </h1>
  </div>
</div>
`;
